tests/integ/plugins/testrunner_test.go

```diff
@@ -132,51 +134,11 @@ func TestSuccessfulCompletion(t *testing.T) {
 		_, err := tr.Run(ctx, &test.Test{TestStepsBundles: testSteps}, targets, jobID, runID, nil)
 		errCh <- err
 	}()
+
 	select {
 	case err = <-errCh:
 		require.NoError(t, err)
 	case <-time.After(successTimeout):
 		t.Errorf("test should return within timeout (%s)", successTimeout.String())
 	}
 }
-
```

Comment:
seems that most is lost in pkg/types; this rename to "plugins" here actually introduces more tests; only had the cmd one, now has exec as well

Reply:
i had another look at this, after removing the Duration from pkg/types; Seems that most of it is lack of a test of the OCP stuff. I didn't add any because that part is still not stable, since we're still discussing the standard. Do you think it's necessary at this point in time?